### PR TITLE
[5.3] Change read to timestamp from boolean

### PR DESCRIPTION
**Pull request by alexbowers**

### CODE REVIEW

**Comment by alexbowers** [comment]
Im unsure if this should be `timestamp` or `datetime` in the stub

File: src/Illuminate/Notifications/DatabaseNotification.php [hunk 1]
```diff
@@ -52,7 +53,7 @@ public function notifiable()
      */
     public function markAsRead()
     {
-        $this->forceFill(['read' => true])->save();
+        $this->forceFill(['read' => Carbon::now()])->save();
```
Comment (dwightwatson) [inline src/Illuminate/Notifications/DatabaseNotification.php]: Just a thought, could do `$this->freshTimestamp()` here instead - would give you a new Carbon instance and save needing to add the use statement.

Comment (alexbowers) [inline src/Illuminate/Notifications/DatabaseNotification.php]: I've never used that before.

Is it done like this:

`$this->forceFill(['read' => $this->freshTimestamp()])->save();`

Comment (dwightwatson) [inline src/Illuminate/Notifications/DatabaseNotification.php]: Yeah - that's the one!

**Comment by a-komarev** [comment]
Maybe it should be named `read_at` to follow consistency of naming timestamp fields?

**Comment by tomschlick** [comment]
+1 for `read_at`

**Comment by taylorotwell** [comment]
Could you send PR to docs when you get a chance. Thanks!

**Comment by alexbowers** [comment]
@taylor I have documented this on the docs branch, but there is a merge conflict, so feel free to take the deltas and just apply them yourself :)

**Pull request merged**